lib/modules/platform/bitbucket/utils.ts

```diff
@@ -181,6 +181,7 @@ export function prInfo(pr: PrResponse): Pr {
     number: pr.id,
     displayNumber: `Pull Request #${pr.id}`,
     bodyStruct: getPrBodyStruct(pr.summary?.raw),
+    body: pr.summary?.raw,
```

Comment:
@zharinov does this work with our PR cache? don't we loose the body when loading PR from cache? 🤔

Reply:
As far as I know, Bitbucket isn't a subject for PR caching

Reply:
but any other platforms? this PR is changing all platforms.

Reply:
I did this because the interface doesn't indicate that this property is only used for one purpose and cannot be used for other purposes. We should either change the property name `body` to something different and only use it for the supported platforms, or all platforms should set the body if applicable IMO. 

BTW: I was also thinking whether we should only add/cache the body when the body contains `<!--renovate:...-->` as this is the only use case for now where it is required.

Maybe there is also another way to get the body of existing pull requests within https://github.com/renovatebot/renovate/pull/19596/files#diff-5b0b84be1a7e3c33085649b328be2ab885d61f872bc82e0a63a505d3e373fe06R331. With that this change wouldn't be necessary.

Reply:
body is at least missing on GitHub when caching is active. so it'll will overwrite manual changes once